Extend Horizontal Bar documentation (#429)
obsidianmd/obsidian-docs: en/How to/Format your notes.md

### Horizontal Bar

> Use three or more stars ***, hyphens ---, or underscores ___ in a new line to produce an horizontal bar. Optionally symbols could be separated by spaces.

Reviewer: ```suggestion
Use three or more stars ***, hyphens ---, or underscores ___ in a new line to produce an horizontal bar. You can also separate symbols using spaces.
```
